content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -150,9 +150,9 @@ the possible following fields:
   (the file's location).
 - `deps`: List of <abbr>dependency</abbr> file or directory paths of this stage
   (relative to `wdir` which defaults to the file's location)
-- `params`: List of [parameter dependencies](/doc/command-reference/params).
-  These are key paths referring to a YAML, JSON or TOML file (`params.yaml` by
-  default).
+- `params`: List of <abbr>parameter<abbr> dependencies. Contains subfields that
+  specify key/value pairs that are read from an arbitrary YAML, JSON, or TOML
```

Comment:
They are field names from an arbitrary ....

Reply:
True. Updated but kept "key/value pairs" because both the name and last value are saved in dvc.yaml.